### DISCUSSION TITLE
Mobile Outlook.com config

### DISCUSSION THREAD

**Discussion by ssaaiidd** [add2cal/add-to-calendar-button#143] [Q&A] [answered]
When I click on Outlook.com on mobile devices, it downloads .ics file. Is there any way to change this? I want it to redirect to outlook.com like on desktop. Thanks.

**Comment by jekuer** [comment, marked as answer]
This is a temporary workaround due to a known bug at Microsoft, where they somehow alter the time on mobile OWA.
So, unfortunately, it would not recommended to change this at the moment.
I will keep you posted. Also feel free to follow the discussion at sh update-tzdata.sh #113
